exporter/prometheus.go

```diff
@@ -0,0 +1,350 @@
+package exporter
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	info "github.com/google/cadvisor/info/v1"
+	"github.com/google/cadvisor/manager"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const namespace = "container"
```

Comment:
You are using this in only one place now. So you could get rid of it. (Or include it in the Desc generation...)